opm/utility/ECLGraph.cpp

```diff
@@ -274,6 +279,19 @@ namespace {
                 std::vector<T>
                 scatterToGlobal(const std::vector<T>& x) const;
 
+                /// Map input vector to active grid cells.
+                ///
+                /// \param[in] x Input vector, defined on the explicitly
+                ///              active cells, all global cells or some
+                ///              other subset (e.g., all non-neighbouring
+                ///              connections).
+                ///
+                /// \return Input vector mapped to active cells or unchanged
```

Comment:
No, that's not the case.  This also handles input vectors of size `SUM(ACTNUM != 0)`  which is, potentially, different from the grid's active cells (`pore_volume > 0`).  We copy the input to the output only if the number of elements is different from both of `numGlobalCells()` and `SUM(ACTNUM != 0)`.